{"name": "test_login[密码错误，登录失败]", "status": "passed", "attachments": [{"name": "stdout", "source": "f701c1e9-74ae-4a67-a882-0ff8b375f7be-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'admin'"}, {"name": "password", "value": "'1234567'"}, {"name": "expect", "value": "{'code': 801, 'message': '用户名或者密码错误'}"}], "start": 1648190890015, "stop": 1648190890026, "uuid": "293bca40-aab9-44d7-86f5-3a05ef7b64f0", "historyId": "c4b9787866ea81ab668752bfda5c4716", "testCaseId": "8bd8079eaae40b2d729c0d112c307024", "fullName": "case.test_login#test_login", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "PC-20200806UOAU"}, {"name": "thread", "value": "78776-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_login"}]}